java/com/google/javascript/jscomp/Diagnostics.java

```diff
@@ -87,12 +89,15 @@
           "duplicate",
           "es5Strict",
           "externsValidation",
+          "extraProvide",
```

Comment:
Looking at history, I think this was excluded on purpose; I don't want to dig why. Let's keep it for now.

Reply:
Please clarify: keep it in the list or add another commit to remove this line?

Reply:
Sorry for typo, I meant let's keep it that way. i.e. Don't include it in the list.